src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -37,17 +43,45 @@ class LicenceConditionService(
     val entityStandardPssConditions = request.standardPssConditions.transformToEntityStandard(licenceEntity, "PSS")
 
     val username = SecurityContextHolder.getContext().authentication.name
+    val createdBy = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
+      ?: throw RuntimeException("Staff with username $username not found")
 
     val updatedLicence = licenceEntity.copy(
       standardConditions = entityStandardLicenceConditions + entityStandardPssConditions,
       dateLastUpdated = LocalDateTime.now(),
       updatedByUsername = username,
     )
+
+    /*
+    When the function is called, the standard conditions are updated in line with the current policy version but
+    the licence will retain the version of the policy it was created with because any additional conditions will be on a previous
+    version and not updated. These additional conditions can only be updated via the vary journey.
+    */
+    val currentPolicyVersion = licencePolicyService.currentPolicy().version
+
+    val changes = mapOf(
+      "typeOfChange" to "update",
+      "condition" to "standard",
+      "changes" to emptyMap<String, Any>()
+    )
```

Comment:
meh, NBD, but could define these closer to where they are used, or even inline the changes payload